src/main/java/RestApi/movies/service/MovieService.java

```diff
@@ -1,7 +1,59 @@
 package RestApi.movies.service;
 
+import RestApi.movies.model.Movies;
+import RestApi.movies.model.Movie;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Component;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 @Component
 public class MovieService {
+
+    ObjectMapper objectMapper = new ObjectMapper();
+
+    public Movies findAll() throws IOException {
+
+        byte[] jsonData = Files.readAllBytes(Paths.get("movies.json"));
```

Comment:
We're currently reading the JSON file for each request. Since the data is static, could we read the data into memory on application boot, using - say - the class constructor?